terraform/iam.tf

```diff
@@ -0,0 +1,44 @@
+#====================================================
+# IAM user, group, policy
+#====================================================
+resource "aws_iam_user" "terraform_user" {
+  name = "terraform-user"
+}
+
+resource "aws_iam_access_key" "tf_user_key" {
+  user = aws_iam_user.terraform_user.name
+}
+
+resource "aws_iam_policy" "terraform_policy" {
+  name = "terraform-policy"
+  path = "/"
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = [
+          "ec2:*",
+          "s3:*",
+          "iam:*",
+          "lambda:*",
+          "route53:*",
+          "application-autoscaling:*",
+          "cloudwatch:*",
+          "dynamodb:*",
+          "ecs:*",
+          "acm:*",
+          "elasticloadbalancing:*",
+          "sns:*"
+        ]
+        Effect   = "Allow"
+        Resource = "*"
```

Comment:
best practice would be to specify the resource a little bit more.

Maybe split the statement also for different services.

But for capstone also fine now :)

Reply:
When I use a sandbox, the resource changes every time, or not?
Will leave it as it is for now, have to look more into the topic.